clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -91,6 +100,14 @@ public class GravitinoVirtualFileSystem extends FileSystem {
   private static final String SLASH = "/";
   private final Map<String, FileSystemProvider> fileSystemProvidersMap = Maps.newHashMap();
 
+  private static final Set<String> CATALOG_NECESSARY_PROPERTIES_TO_KEEP =
+      Sets.newHashSet(
+          OSSProperties.GRAVITINO_OSS_ENDPOINT,
```

Comment:
seems missing xxx_REGION?

Reply:
I'm not very clear where xxx_region is used and the current code also works, could you please provide more detail about the necessity for this configuration.

Reply:
if user doesn't specify S3 endpoint, I doubt whether it could works without region. For OSS, seems doesn't need region because OSS endpoint is required. BTW there are duplicated key in the SET.

Reply:
I see.